EShop.Web/Controllers/ProductsController.cs

```diff
@@ -6,7 +6,7 @@ namespace EShop.Web.Controllers
 {
     public class ProductsController : Controller
     {     
-        private readonly IRepository<ProductModel> productRepository;
+        private  IRepository<ProductModel> productRepository;
```

Comment:
Nu vad vre-un motiv sa scoti readonly de pe `productRepository`, nu faci re-assign (productRepository = somethingOther) nicaieri in cod, deci ar fi mai ok sa ramana readonly.